apps/coscrad-frontend/src/components/commands/command-panel.tsx

```diff
@@ -69,7 +74,7 @@ export const CommandPanel = ({ actions, commandContext }: CommandPanelProps) =>
 
     return (
         <CommandWorkspace
-            selectedCommand={selectedCommand}
+            SelectedForm={selectedCommand}
```

Comment:
Is this the correct name for this prop?

Reply:
I did some renaming that more accurately reflects the executor \ form distinction.